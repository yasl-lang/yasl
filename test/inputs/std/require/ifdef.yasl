const m = require('test/inputs/math.yam')

echo m

ifdef math {
    echo 'script: math is defined'
} else {
    echo 'script: math is not defined'
}
