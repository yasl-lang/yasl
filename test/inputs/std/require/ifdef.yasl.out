submodule: math is defined
script: math is defined
